src/main/java/alexiil/mods/load/json/ImageRender.java

```diff
@@ -24,7 +24,8 @@ public ImageRender(String resourceLocation, EPosition positionType, EType type,
     }
 
     public int transformX(int screenWidth) {
-        return positionType.transformX(position.x, screenWidth - position.width);
+        int realX=position.width==0?0:screenWidth - position.width;
+        return positionType.transformX(position.x, realX;
```

Comment:
Add a close bracket here.
(I just looked at the travis build for the code error)